src/together/types/files.py

```diff
@@ -43,21 +48,14 @@ class FileResponse(BaseModel):
     Files API response type
     """
 
-    # file id
```

Comment:
The class fields are self-explanatory, and redundant comments are usually a poor practice due to their additional maintenance cost with little value. I kept a few comments for less obvious fields and removed the rest